webrtc.html

```diff
@@ -2481,7 +2519,17 @@
             associated, via <code>setLocalDescription</code>,
             with this <code><a>RTCRtpSender</a></code> object.
           </dd>
-        </dl>
+
+	  <dt>attribute PriorityValue priority</dt>
```

Comment:
I think this is not the right place for this attribute. As a per-ssrc attribute, it should be on RtpEncodingParams (which haven't landed in the spec yet).